example_data/sample_bundle.json

```diff
@@ -34,72 +49,75 @@
         "resourceType": "Patient",
-        "id": "37176e84-d977-e993-3c49-d76fcfc6e625",
+        "id": "gosh-redcap-40388914",
         "active": true,
         "name": [
           {
-            "family": "Manager",
+            "family": "BrighGosh321",
             "given": [
-              "General"
+              "Maria321"
             ]
           }
         ],
-        "gender": "female",
-        "birthDate": "2016-06-01",
-        "managingOrganization": {
-          "reference": "Organization/85940927-468f-f53d-864a-7a50b48d73f1"
+        "gender": "male",
+        "birthDate": "2019-08-02",
+        "text": {
+          "status": "generated",
+          "div": "<div xmlns=\\\"http://www.w3.org/1999/xhtml\\\">Maria321 BrighGosh321 from RedCap FHIR</div>"
         },
         "identifier": [
           {
-            "value": "40057119"
+            "use": "official",
+            "assigner": {
+              "display": "GOSH MRN"
+            },
+            "system": "https://fhir.nhs.uk/Id/mrn",
+            "value": "40388914"
           },
           {
-            "extension": [
-              {
-                "url": "https://fhir.nhs.uk/R4/StructureDefinition/Extension-UKCore-NHSNumberVerificationStatus",
-                "valueCodeableConcept": {
-                  "coding": [
-                    {
-                      "system": "https://fhir.nhs.uk/R4/CodeSystem/UKCore-NHSNumberVerificationStatus",
-                      "code": "Z60791509",
-                      "display": "Family Number"
-                    }
-                  ]
-                }
-              }
-            ]
+            "system": "https://fhir.nhs.uk/Id/nhs-number",
+            "value": "8105688202.0"
+          },
+          {
+            "system": "https://fhir.nhs.uk/Id/nhs-family-number",
+            "value": "Z968769"
           }
         ],
-        "generalPractitioner": {
-          "type": "Practitioner",
-          "reference": "Practitioner/d3447490-96fd-35d0-adf2-0806e5214606"
+        "managingOrganization": {
+          "type": "Organization",
+          "reference": "Organization?_id=gosh-redcap-20RG-205G0047"
         }
       },
       "request": {
-        "method": "POST",
-        "url": "Patient"
+        "method": "PUT",
+        "url": "Patient?_id=gosh-redcap-40388914"
       }
     },
     {
-      "fullUrl": "Specimen/fb82860d-eabc-a8d0-b341-facdff0ac0f1",
+      "fullUrl": "Specimen/specimen-20RG-205G0047",
       "resource": {
         "resourceType": "Specimen",
-        "id": "fb82860d-eabc-a8d0-b341-facdff0ac0f1",
-        "receivedTime": "2020-03-12T14:15:00",
+        "id": "specimen-20RG-205G0047",
+        "receivedTime": "2020-07-23T15:19:00",
         "collection": {
-          "collectedDateTime": "2020-06-12T14:15:00"
+          "collectedDateTime": "2020-07-23T12:22:00"
         },
+        "processing": [
+          {
+            "timeDateTime": "2020-10-21T15:00:00"
+          }
+        ],
```

Comment:
this is new, what does the processing datetime map to? And does that mean we're removing it from elsewhere?

Reply:
Yes, it is new. it's mapped to the "specimen authorized time"